src/services/query.ts

```diff
@@ -30,6 +31,7 @@ export const buildResourceQuery = (
     ...defaultQueryParams,
     resource,
     ...queryParamsOverrides,
+    datasource: { uid: VAR_DATASOURCE_EXPR } as DataSourceApi,
```

Comment:
Interesting, I think the type in `LokiQuery` should have been:

```
datasource?: DataSourceRef;
```

Mind changing that so you don't need the type cast here.